File: 241006/조건에 따른 연산/operatino-by-rule.py
n = int(input())
cnt = 0

while True:

    cnt += 1

    if n % 2 == 0:
        n = n * 3 + 1
    else:
        n = n * 2 + 2

    if n >= 1000:
        break
    

print(cnt)

File: 241006/조건에 따른 연산/operatino-by-rule.js
const fs = require("fs");
let n = Number(fs.readFileSync(0).toString().trim());

let cnt = 0;

while (true) {

    if (n >= 1000) {
        break;
    }
    if (n % 2 === 0) {
        n = n * 3 + 1;
    }
    else {
        n = n * 2 + 2;
    }

    cnt++
}

console.log(cnt);